lib/alpha_taxonomy/taxon_linker.rb

```diff
@@ -0,0 +1,23 @@
+require "csv"
+
+module AlphaTaxonomy
+  class TaxonLinker
+    def run!
+      ImportFile.new.grouped_mappings.each do |base_path, taxon_titles|
+        taxon_content_ids = taxon_titles.map do |taxon_title|
+          taxon_base_path = TaxonPresenter.new(title: taxon_title).base_path
+          taxon_content_item = Services.content_store.content_item!(taxon_base_path)
+          taxon_content_item["content_id"]
+        end
+        target_content_item = Services.content_store.content_item!(base_path)
```

Comment:
re: 404 - it fails noisily. This should probably be re-worked to report a list of failed paths at the end of the run.

Good suggestion re: `ContentLookForm`, will revise.